java/code/src/com/redhat/rhn/taskomatic/task/SSHService.java

```diff
@@ -14,26 +14,26 @@
  */
 package com.redhat.rhn.taskomatic.task;
 
-import com.redhat.rhn.taskomatic.task.sshpush.SSHPushDriver;
+import com.redhat.rhn.taskomatic.task.sshservice.SSHServiceDriver;
 
 /**
- * Call rhn_check on relevant systems via SSH using remote port forwarding.
+ * Provide services for salt ssh clients
  */
-public class SSHPush extends RhnQueueJob<SSHPushDriver> {
+public class SSHService extends RhnQueueJob<SSHServiceDriver> {
 
-    public static final String QUEUE_NAME = "ssh_push";
+    public static final String QUEUE_NAME = "ssh_service";
 
     @Override
     public String getConfigNamespace() {
-        return "sshpush";
+        return "sshservice";
```

Comment:
Is it safe to change this without fallback? It's not exactly documented, but it's possible to set custom values for `taskomatic.sshpush.parallel_threads` and `taskomatic.sshpush.reschedule_time`

Reply:
This would be already the 2nd fallback. I think for 5.0 we can rename this. I will check if we have an issue already where we can add a list for things which needs to go into the release notes.

Reply:
https://github.com/SUSE/spacewalk